docs: add release notes for 0.19.6 (#5357)
determined-ai/determined: docs/release-notes.rst

 Version 0.19
**************

Version 0.19.6
==============

**Release Date:** October 28, 2022

**Breaking Changes**

> -  API: removed legacy endpoint ``/tasks/:task_id`` due to it always incorrectly returning missing

Reviewer: ```suggestion
-  API: Remove the legacy endpoint ``/tasks/:task_id`` due to it always incorrectly returning missing
```

>    parameter.

Reviewer: "missing parameters"? "a missing parameter"? Not sure what this is describing.

> -  Additional Slurm options formerly specified in the experiment environment section are now part of

Reviewer: ```suggestion
-  Experiment: Additional Slurm options formerly specified in the experiment environment section are now part of
```

>    a new Slurm section of the experiment configuration. For example, what was formerly written as:

Reviewer: ```suggestion
   a new ``slurm`` section of the experiment configuration. For example, what was formerly written as
```

   .. code:: yaml

      environment:
      ...
        slurm:
          - --mem-per-cpu=10
          - --exclusive

>    Is now specified as:

Reviewer: ```suggestion
   is now specified as
```

   .. code:: yaml

      environment:
      ...
      slurm:
        sbatch_args:
          - --mem-per-cpu=10
          - --exclusive

**Improvements**

> -  CLI: Add the `ls` abbreviation for `list` to all applicable CLI commands.

Reviewer: ```suggestion
-  CLI: Add the ``ls`` abbreviation for ``list`` to all applicable CLI commands.
```

> -  Support a new ``-i``/``--include`` option in task-starting CLI commands. The context option

Reviewer: ```suggestion
-  CLI: Support a new ``-i``/``--include`` option in task-starting CLI commands. The context option
```

   (``--context``) is useful for copying a directory of files into the task container, but it may

>    only be provided once, and it can be clunky if you only care about one or two files. The Include

Reviewer: ```suggestion
   only be provided once, and it can be clunky if you only care about one or two files. The ``--include``
```

   option also copies files into the task container, but:

>       -  The directory name is preserved, so ``-i my_data/`` would result in a directory named

Reviewer: Dedent this list.

         ``my_data/`` appearing in the working directory of the task container.

>       -  It may point to a file, so ``-i my_data.csv`` will work.

Reviewer: ```suggestion
      -  It may point to a file, so ``-i my_data.csv`` will place ``my_data.csv`` into the working directory.
```
